### ISSUE TITLE
Stream key

### ISSUE THREAD

**Issue by kasyr** [PonnamKarthik/FlutterRtmpPlublisher#8]
I am trying to broadcast a stream with Mux, to make it work I need to use both url and stream key. Is it possible to use a stream key with this library?

**Comment by alex-heritier** [comment]
Any follow up on this? Facing the same issue.

**Comment by alex-heritier** [comment]
I got around this by appending the stream key to the end of the RTMP url like so 
Example: rtmp://global-live.mux.com:5222/app/STREAM_KEY

**Comment by pojoba02** [comment]
@alex-heritier is the stream key in mux the same as the id from URL Signing Keys.

**Comment by alex-heritier** [comment]
> @alex-heritier is the stream key in mux the same as the id from URL Signing Keys.

No, this explains Mux stream keys https://docs.mux.com/docs/live-streaming

**Comment by kasyr** [comment]
> I got around this by appending the stream key to the end of the RTMP url like so
> Example: rtmp://global-live.mux.com:5222/app/STREAM_KEY

Thank you! I got it working :)

**Comment by pojoba02** [comment]
@alex-heritier and @kasyr did you guys have some data on mux dashboard, cos mine keep saying connecting and never does.

**Comment by ollyde** [comment]
@pojoba02 same issue, also the UI is broken on iOS. Would be nice if this was just a view.

**Comment by pojoba02** [comment]
@ollydixon is pinkfish/flutter_rtmppublisher well a shot, it seem he have nothing to do with rtmp url.

**Comment by ollyde** [comment]
@pojoba02 yes I got that one working :-)

**Comment by pojoba02** [comment]
android version!

**Comment by pojoba02** [comment]
do you have to connect it to rtmp url?

**Comment by ollyde** [comment]
@pojoba02 yes.

**Comment by pojoba02** [comment]
ok cool

**Comment by pojoba02** [comment]
just run the iOS version, pretty pretty cool plugin,
